src/main/webapp/app/shared/layouts/navbar/navbar.component.html

```diff
@@ -172,11 +172,13 @@
                     <li>
                         <h6 class="dropdown-header" jhiTranslate="global.menu.language">Language</h6>
                     </li>
-                    <li *ngFor="let language of languages">
-                        <a class="dropdown-item" [jhiActiveMenu]="language" href="javascript:void(0);" (click)="changeLanguage(language); collapseNavbar()">{{
-                            language | findLanguageFromKey
-                        }}</a>
-                    </li>
+                    <div *ngIf="languages && languages.length > 1">
+                        <li *ngFor="let language of languages">
+                            <a class="dropdown-item" [jhiActiveMenu]="language" href="javascript:void(0);" (click)="changeLanguage(language); collapseNavbar()">{{
```

Comment:
Why anchor if it does not go anywhere? javascript:void(0) violates Content Security Policy they say :D

Reply:
I didn't change that part just added a `div` with an `ngIf`.